mllib/src/test/scala/org/apache/spark/ml/feature/QuantileDiscretizerSuite.scala

```diff
@@ -99,33 +73,3 @@ class QuantileDiscretizerSuite
     testDefaultReadWrite(t)
   }
 }
-
-private object QuantileDiscretizerSuite extends SparkFunSuite {
-
-  def checkDiscretizedData(
-      sc: SparkContext,
-      data: Array[Double],
-      numBucket: Int,
-      expectedResult: Array[Double],
-      expectedAttrs: Array[String]): Unit = {
-    val sqlCtx = SQLContext.getOrCreate(sc)
-    import sqlCtx.implicits._
-
-    val df = sc.parallelize(data.map(Tuple1.apply)).toDF("input")
-    val discretizer = new QuantileDiscretizer().setInputCol("input").setOutputCol("result")
-      .setNumBuckets(numBucket).setSeed(1)
-    val model = discretizer.fit(df)
-    assert(model.hasParent)
```

Comment:
The `assert(model.hasParent)` test was added as part of #11214, see https://github.com/apache/spark/pull/11214/files#diff-22d5d4af1af0a7c9d606c5d82b5743f4R97 for reference. Please could we include this test in the new tests.